tests/internal/src/static_analysis/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#####################################
```

Comment:
Thanks for adding an example.  

Could you please add a host-config file that sets up clang-query and runs this?

Reply:
Done